### PR TITLE
#584: Added User-Agent header with version

### PR DESCRIPTION
**Pull request by bonkey**
#584: Added User-Agent header with version

### CODE REVIEW

File: AtlasSDK/AtlasSDK/Utils/Networking/URLRequest.swift [hunk 1]
```diff
@@ -66,4 +65,16 @@ extension URLRequest {
...
+    fileprivate func setContentType(for endpoint: Endpoint, headers: inout [String: String]) {
+        let acceptedContentTypes = [endpoint.acceptedContentType, "application/x.problem+json"].flatMap { $0 }
+        headers["Accept"] = acceptedContentTypes.joined(separator: ",")
+        headers["Content-Type"] = endpoint.contentType
+    }
+
+    fileprivate func setUserAgent(headers: inout [String: String]) {
+        let bundle = Bundle(for: RFC3339DateFormatter.self)
+        let version = bundle.string(for: "CFBundleVersion")
+        headers["User-Agent"] = "AtlasSDK v.\(version~?)"
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Utils/Networking/URLRequest.swift]: probably `Atlas iOS SDK` as we have JS SDK now?

Comment (bonkey) [inline AtlasSDK/AtlasSDK/Utils/Networking/URLRequest.swift]: Well, true. Should be there iOS version as well

Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Utils/Networking/URLRequest.swift]: And device type?

**Comment by bonkey** [comment]
👍

**Pull request merged**